x-pack/plugins/lens/public/embeddable/embeddable.tsx

```diff
@@ -1088,6 +1107,7 @@ export class Embeddable
                 embeddableTitle: this.getTitle(),
                 ...(input.palette ? { theme: { palette: input.palette } } : {}),
                 ...('overrides' in input ? { overrides: input.overrides } : {}),
+                ...getInternalTables(this.savedVis.state.datasourceStates),
```

Comment:
we extract all tables from layers and pass them as variables to expressions, so expressions can later access them using `var` function